packages/headless/src/controllers/commerce/facets/core/headless-core-commerce-facet.test.ts

```diff
@@ -199,21 +181,12 @@ describe('CoreCommerceFacet', () => {
     describe('when toggled facet value state is "idle"', () => {
       const facetValue = () =>
         buildMockCommerceRegularFacetValue({value: 'TED', state: 'idle'});
-      it('dispatches #toggleExcludeActionCreator with the passed facet value', () => {
-        facet.toggleSingleExclude(facetValue());
 
-        expect(engine.actions).toContainEqual(
-          toggleExcludeActionCreator({facetId, selection: facetValue()})
-        );
-      });
-
-      it('dispatches #fetchResultsActionCreator', () => {
+      it('calls #toggleExclude', () => {
```

Comment:
@Spuffynism I decided to take the same approach as you did here https://github.com/coveo/ui-kit/pull/3461/files#diff-1e116ab2a783896b6c34ce328580a6ebdf37cc4922848a6efb5de3e2c59898bfR155 for testing #toggleSingleSelect and #toggleSingleExclude (i.e., testing that it calls a method instead of testing the actions that are dispatched by the called method).